_beats/libbeat/docs/version.asciidoc

```diff
@@ -1,4 +1,4 @@
-:stack-version: 7.0.0-alpha1
+:stack-version: 6.1.0
```

Comment:
I assume this is on purpose? So in the docs we already link to the `6.1.0` version although we are in the `6.0` version right?

Reply:
We could do that in the docs build if we want, but for now there is only a docs build for master. The docs build is independent of the unified release build.

Reply:
This means that this version here does not cause anything, because docs are only built in master?
Just wanted to ensure that it is on purpose to point to `6.1.0` here instead of `6.0.x`.

Reply:
Currently for apm-server we only build the master branch: https://github.com/elastic/docs/blob/master/conf.yaml#L728 Assuming we would add the 6.0 branch for building, the doc build would still work but all links would point to 6.1. 

The reason it points to 6.1 is because we also use the 6.x beats branch here. From a docs perspective it's not "correct" but as I don't think we plan to do a doc build based on 6.0 I would not fix this issue.

Reply:
ok, thx for additional explanation